tools/tsp-client/README.md

```diff
@@ -0,0 +1,46 @@
+# tsp-client
+
+A simple command line tool for generating TypeSpec clients.
```

Comment:
Does this tool target to be consumed in CI pipeline?

Reply:
I would hope it does get used by any automation that needs to do these workflows.

Reply:
Yeah, it should work as part of a CI pipeline, not sure if we have any particular requirements for CLI tools to make them more CI friendly, but happy to adjust anything we need.